metals/src/main/resources/db/migration/V4__Fingerprints.sql

```diff
@@ -0,0 +1,7 @@
+-- Fingerprints saved between invocations
+create table fingerprints(
+  path varchar not null,
+  text varchar not null,
+  md5 varchar not null,
```

Comment:
Why do we need both base64 and md5 encoded content? (if I understand correctly)
I guess there's no point encrypting the content with md5, since now we have base64 encoded one?

Reply:
Maybe this is because we are using Md5Fingerprint class? Maybe we should refactor in another PR not to use md5 🤔

Reply:
Maybe base64 is no needed in reality, just felt like it's a bit cleaner to save it this way. I can revert that.

As for MD5, we still need it since that is much better suited for a hash, base64 can be quite long.

Reply:
Base64 is just an encoding which size is probably much longer than original text, where md5 is a hash function which converts content to always (I think) 16 bytes.

PS. Base64 is not an encryption :D

Reply:
> Base64 is just an encoding which size is probably much longer than original text
Somehow I thought that it's shorter, only after thinking on this a bit a realised that was not a great idea 😅

Reply:
About length - https://stackoverflow.com/a/13378842

Reply:
Ah, if we don't need to decode to the original text, we can keep md5 :)
I was mentioning md5 as encrypt, not base64

Reply:
Removed the usage of Base64. It was needed at all indeed.